### PR TITLE
Feat/cuda lb

### PR DESCRIPTION
**Pull request by S1ro1**
## Description
Cuda leaderboards cc @alexzhang13 

## Checklist

Before submitting this PR, ensure the following steps have been completed:

- [ ] Run the slash command `/verifyruns` on your own server.
  - Run the cluster bot on your server:
    ```bash
    python discord-bot.py
    ```
  - Start training runs with the slash command `/verifyruns`.
  - Verify that the bot eventually responds with:
    ```
    ✅ All runs completed successfully!
    ```
    (It may take a few minutes for all runs to finish. In particular, the GitHub
    runs may take a little longer. The Modal run is typically quick.)
  For more information on running a cluster bot on your own server, see
  README.md.

### CODE REVIEW

File: src/discord-cluster-manager/leaderboard_db.py
Comment (alexzhang13) [inline src/discord-cluster-manager/leaderboard_db.py:1]: Let's put this change in a separate commit so it's not confusing.

**Review by alexzhang13 — CHANGES_REQUESTED** [review]
@S1ro1 The .cu eval is fixed, I'm just being a little nitpicky but we should move that db connection change to a different PR for clarity sake.

**Comment by S1ro1** [comment]
> @S1ro1 The .cu eval is fixed, I'm just being a little nitpicky but we should move that db connection change to a different PR for clarity sake.

Those changes shouldn't even be there, I just don't have ssl running on my local db so I have to change this everytime I dev locally and I probably commit by mistake

**Review by alexzhang13 — APPROVED** [review]
It's working now for basic kernels -- arbitrary names we'll double check in follow up PR
